### DISCUSSION TITLE
HW#5 V5 Blank screen on read rom for SNES and N64

### DISCUSSION THREAD

**Discussion by rinemetal** [sanni/cartreader#1000] [Need help]
Hi, I just finished up my V5 and everything works as it should for the top readers I have tested (famicom and genesis) along with the Gameboy slot. Those slots work as they should and store roms and save files without issue. 

I have run into an issue with the SNES and N64 cartridge slots. When selecting read rom (cartridge), it displays a blank screen and hangs until I reset using the on off switch. This occurs both with a cartridge installed and out of the slot. 

https://github.com/sanni/cartreader/discussions/454

I came across the above thread and seen some discussion on the SD cards. I have tried two different SD cards and formatted both in windows10 and using the suggested SD Card Formatter application listed under the wiki. No success. I have also left the unit on for over an hour to see if it was some type of slow loading issues causing the hang, but it never recovered. 

So I believe there may be a hardware issue. Is there a common component between the SNES and N64 that would be causing the same issue on both slots?  This might narrow components to investigate.

**Comment by sanni** [comment]
It could be either the Arduino running out of memory or an incompatible SD card.
If you have enabled any additional systems, try with just the default selection in config.h.
It's best to stick with cards 32GB or less.

Other than that I can't think of anything preventing the OSCR from creating the ROM file on the SD card.
Did you also test GBA or just GB?

**Reply by rinemetal** [reply]
I was hoping this would do the trick, or be a simple software fix but no luck. More info on the hardware troubleshooting in handles888’s reply.

**Comment by handles888** [comment]
Check your clock gen.  Make sure you didn’t solder the pins so it’s upside down.  The side with the electronics should be facing the ceiling.

if memory serves correct the N64 doesn’t use it and you can try to just pull it out and see if it reads the cart.  test to see if it’s faulty even if it was put together properly

**Reply by handles888** [reply]
I saw the response in my email, but I don’t see it in the thread here. The pictures you attached are the CIC not the clock gen.  Check the purple component make sure its not upside down.  Remove it and try to dump N64
![IMG_1162](https://github.com/user-attachments/assets/24fd3778-f154-47dc-b16e-2f9abc6b8e8f)

**Reply by rinemetal** [reply]
Thanks! I went to double check my trace again and edited my post until I could verify. In the process I started checking continuity. I was about to post back up, but you beat me to it! Yeah it is an upside down clock gen! Appreciate the help and jumping on the question. I’ll get it flipped tomorrow and post back.

**Reply by handles888** [reply]
Remove one pin at a time so you don’t pull any pads.

**Reply by rinemetal** [reply]
Sometimes it’s the simple stuff. This is a really cool kit, appreciate the help in completing it.

![IMG_0270](https://github.com/user-attachments/assets/24acbc3f-97ed-4965-a938-b5e36c360be3)
